libindy-crypto/src/bls/mod.rs

```diff
@@ -390,20 +440,27 @@ impl Bls {
         // the C API. Verifiers can thus cache the aggregated verkey and avoid several EC point additions.
         // The code below should be moved to such method.
 
-        let msg_hash = Bls::_hash(message)?;
+        let msg_hash = Bls::_hash_sha2(message)?;
 
         let lhs = Pair::pair(&multi_sig.point, &gen.point)?;
         let rhs = Pair::pair(&msg_hash, &aggregated_verkey)?;
 
         Ok(lhs.eq(&rhs))
     }
 
-    fn _hash(message: &[u8]) -> Result<PointG1, IndyCryptoError> {
+    fn _hash_sha2(message: &[u8]) -> Result<PointG1, IndyCryptoError> {
         let mut hasher = Sha256::default();
         hasher.input(message);
 
         Ok(PointG1::from_hash(hasher.result().as_slice())?)
     }
+
+    fn _hash_sha3(message: &[u8]) -> Result<PointG1, IndyCryptoError> {
+        let mut hasher = Keccak256::default();
```

Comment:
Can you have a single function with logic and make `Keccak256` and `Sha256` as possible args